streams/src/main/java/org/apache/kafka/streams/processor/internals/Task.java

```diff
@@ -71,7 +72,6 @@
         RESTORING(2, 3, 4),    // 1
         RUNNING(3, 4),         // 2
         SUSPENDED(1, 4),       // 3
-        CLOSING(4, 5),         // 4, we allow CLOSING to transit to itself to make close idempotent
```

Comment:
Removing this state -- this is an open question if I did this correctly. \cc @vvcephei